public/api-v1.php

```diff
@@ -158,7 +158,7 @@
     if (array_key_exists('variables',$patch_data)) {
         $scope = new \Tancredi\Entity\Scope($mac, $this->storage, $this->logger);
         $scope->setVariables($patch_data['variables']);
-        $response = $response->withStatus(204);
+        $response = $response->withStatus(200);
```

Comment:
Here and below (line R347), you ought to return the full scope as response body.
```suggestion
        $response = $response->withJson(\Tancredi\Entity\Scope::getPhoneScope($mac, $this->storage, $this->logger), 200, JSON_FLAGS);
```